rubicon/objc/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.2.3'
+__version__ = '0.2.5'
```

Comment:
What happened to 0.2.4 though? I see it's on https://pypi.python.org/pypi/rubicon-objc, but in the repo it existed for two minutes before being replaced with 0.2.5.